lib/ae_mdw/db/name.ex

```diff
@@ -29,6 +29,7 @@ defmodule AeMdw.Db.Name do
   @typep height :: Blocks.height()
   @typep txi_idx :: AeMdw.Txs.txi_idx()
   @typep state :: State.t()
+  @typep pointer() :: %{key: binary(), id: binary()}
```

Comment:
This type seems to differ from the openapi specs (the `encoded_key` is missing)

Reply:
Done, thanks